packages/synapse-react-client/package.json

```diff
@@ -38,20 +38,20 @@
     "@brainhubeu/react-carousel": "1.19.26",
     "@emotion/react": "^11.14.0",
     "@emotion/styled": "^11.14.0",
-    "@mui/icons-material": "^5.17.1",
-    "@mui/material": "^5.17.1",
-    "@mui/system": "^5.17.1",
-    "@mui/utils": "^5.17.1",
-    "@mui/x-data-grid": "^6.20.4",
-    "@mui/x-date-pickers": "^6.20.2",
+    "@mui/icons-material": "^7.1.0",
+    "@mui/material": "^7.1.0",
+    "@mui/system": "^7.1.0",
+    "@mui/utils": "^7.1.0",
+    "@mui/x-data-grid": "^8.4.0",
+    "@mui/x-date-pickers": "^8.4.0",
     "@popperjs/core": "^2.11.8",
     "@react-google-maps/api": "^2.20.6",
     "@react-hook/resize-observer": "^1.2.6",
     "@react-hookz/web": "^23.1.0",
-    "@rjsf/core": "5.24.10",
-    "@rjsf/mui": "5.24.10",
-    "@rjsf/utils": "5.24.10",
-    "@rjsf/validator-ajv8": "5.24.10",
+    "@rjsf/core": "6.0.0-beta.10",
+    "@rjsf/mui": "6.0.0-beta.10",
+    "@rjsf/utils": "6.0.0-beta.10",
+    "@rjsf/validator-ajv8": "6.0.0-beta.10",
```

Comment:
RJSF v6 adds support for MUI v7. It is still in beta, but I don't expect any additional significant breaking changes between beta.10 and the full v6 release